[2025-04-09T15:25:50.266+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T15:25:50.284+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.load_raw_data manual__2025-04-09T15:25:43.596840+00:00 [queued]>
[2025-04-09T15:25:50.292+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.load_raw_data manual__2025-04-09T15:25:43.596840+00:00 [queued]>
[2025-04-09T15:25:50.293+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-09T15:25:50.393+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): load_raw_data> on 2025-04-09 15:25:43.596840+00:00
[2025-04-09T15:25:50.400+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=271) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-09T15:25:50.402+0000] {standard_task_runner.py:72} INFO - Started process 273 to run task
[2025-04-09T15:25:50.402+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'load_raw_data', 'manual__2025-04-09T15:25:43.596840+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmplp7k58hj']
[2025-04-09T15:25:50.404+0000] {standard_task_runner.py:105} INFO - Job 78: Subtask load_raw_data
[2025-04-09T15:25:50.505+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.load_raw_data manual__2025-04-09T15:25:43.596840+00:00 [running]> on host 5a8aeda502d0
[2025-04-09T15:25:50.647+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='load_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T15:25:43.596840+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T15:25:43.596840+00:00'
[2025-04-09T15:25:50.648+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-09T15:25:50.661+0000] {sql.py:266} INFO - Executing: 
    COPY driven_raw.raw_batch_data(
    person_name, user_name, email, personal_number, birth_date,
    address, phone, mac_address, ip_address, iban, accessed_at,
    session_duration, download_speed, upload_speed, consumed_traffic, unique_id
    ) 
    FROM '/opt/***/data/raw_data.csv' 
    DELIMITER ',' 
    CSV HEADER;
    
[2025-04-09T15:25:50.669+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-09T15:25:50.680+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-09T15:25:50.683+0000] {sql.py:509} INFO - Running statement: 
    COPY driven_raw.raw_batch_data(
    person_name, user_name, email, personal_number, birth_date,
    address, phone, mac_address, ip_address, iban, accessed_at,
    session_duration, download_speed, upload_speed, consumed_traffic, unique_id
    ) 
    FROM '/opt/***/data/raw_data.csv' 
    DELIMITER ',' 
    CSV HEADER;
    , parameters: None
[2025-04-09T15:25:50.738+0000] {sql.py:518} INFO - Rows affected: 5712
[2025-04-09T15:25:50.964+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-09T15:25:50.965+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=load_raw_data, run_id=manual__2025-04-09T15:25:43.596840+00:00, execution_date=20250409T152543, start_date=20250409T152550, end_date=20250409T152550
[2025-04-09T15:25:51.021+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T15:25:51.047+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T15:25:51.071+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
